src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindEmbeddingProvidersCommand.java

```diff
@@ -20,9 +20,8 @@ public record Options(
       @Nullable
           @Schema(
               description =
-                  "Filters the models based on their support status, defaults to SUPPORTED to only return models that can be used when creating a new Collection or Table. Supported filter values are SUPPORTED, DEPRECATED, and END_OF_LIFE. Set to null or empty string to return all models.",
+                  "Filters the models based on their support status, only SUPPORTED models will be returned if leaving out the options, these SUPPORTED models can be used when creating a new Collection or Table. Supported filter values are SUPPORTED, DEPRECATED, and END_OF_LIFE. Set to null or empty string to return all models.",
```

Comment:
Should probably mention that "filtering" here means "include in response" (filter in, not out). Not sure of best wording.